query-node/manifest.yml

```diff
@@ -2,37 +2,41 @@ version: '0.1'
 description: Joystream query-node manifest file for sumer
 repository: https://github.com/Joystream/joystream
 dataSource:
-  kind: substrate 
+  kind: substrate
   chain: joystream
   indexerVersion: '0.1.6'
 entities:
   - mappings/lib/generated/**/*.model.js
-metadata:
-  # TODO: update source and blockHash for sumer release
-  source: wss://babylon-sub-1.joystream.app/staging/rpc
-  blockHash: '0x206a6cca31333cb19ee120cde7455e13eea368ec3f6176d0cd48698228410489'
-events:
-  - members.MemberRegistered
-  - members.MemberUpdatedAboutText
-  - members.MemberUpdatedAvatar
-  - members.MemberUpdatedHandle
-  - members.MemberSetRootAccount
-  - members.MemberSetControllerAccount
-calls:
-  - members.buyMembership
-  - members.addScreenedMember
-outDir: ./mappings/generated/types
-customTypes:
-  lib: '@joystream/types/augment/all/types'
-  typedefsLoc: '../types/augment/all/defs.json'
+typegen:
+  metadata:
+    source: wss://sumer.joystream.app
+  events:
+    - members.MemberRegistered
+    - members.MemberUpdatedAboutText
+    - members.MemberUpdatedAvatar
+    - members.MemberUpdatedHandle
+    - members.MemberSetRootAccount
+    - members.MemberSetControllerAccount
+  calls:
+    - members.buyMembership
+    - members.addScreenedMember
+    - members.changeMemberAboutText
+    - members.changeMemberAvatar
+    - members.changeMemberHandle
+    - members.setRootAccount
+    - members.setControllerAccount
+  outDir: ./generated/types
+  customTypes:
+    lib: '@joystream/types/augment/all/types'
+    typedefsLoc: '../types/augment/all/defs.json'
 mappings:
   hydraCommonVersion: '0.0.3'
-    # process only blocks with height >= 1M 
+    # process only blocks with height >= 1M
   blockInterval: '[1000000,]'
```

Comment:
I think `blockInterval` is not need here as we're going to run all the blocks from zero height?

Reply:
`blockInterval` has been removed